openedx/core/djangoapps/theming/helpers.py

```diff
@@ -288,6 +288,18 @@ def get_theme_base_dirs():
     if not is_comprehensive_theming_enabled():
         return []
 
+    # Legacy code for COMPREHENSIVE_THEME_DIR backward compatibility
+    if hasattr(settings, "COMPREHENSIVE_THEME_DIR"):
+        theme_dir = settings.COMPREHENSIVE_THEME_DIR
```

Comment:
I think @fredsmith presented a similar thought in https://github.com/edx/configuration/pull/3099, but it seems to me that if a COMPREHENSIVE_THEME_DIR has been specified, we could simply add (prepend?) it to the list of COMPREHENSIVE_THEME_DIRS instead of making it an either/or situation.